src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,218 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
+
+Derivatives can be stored/distributed in two ways:
+
+1.  Under a `derivatives/` subfolder in the root of the source BIDS dataset
+    folder to make a clear distinction between raw data and results of data
+    processing. Each pipeline has a dedicated directory under which it stores
+    all of its outputs. There is no restriction on the directory name; however,
+    it is RECOMMENDED to use the format `<pipeline>-<variant>` in cases where it
+    is anticipated that the same pipeline will output more than one variant
+    (e.g., `AFNI-blurring`, `AFNI-noblurring`, etc.).
+
+    For example:
+
+    ```Text
+    <dataset>/derivatives/fmripreprocess_01/sub-0001
+    <dataset>/derivatives/spm/sub-0001
+    <dataset>/derivatives/vbm/sub-0001
+    ```
+
+1.  As a standalone dataset independent of the source (raw or derived) BIDS
+    dataset. This way of specifying derivatives is particularly useful when the
+    source dataset is provided with read-only access, and for publishing
+    derivatives as independent bodies of work, or for describing derivatives
+    that were created from more than one source dataset.
+
+The rest of the Derivatives specification assumes Case 1, but Case 2 applies
+after removing `/derivatives/<pipeline_name>` from template names. In both cases
+every derivatives dataset is considered a BIDS dataset and must include a
+`dataset_description.json` file at the root level. Consequently, files should be
+organized to comply with the BIDS to the full extent possible (i.e., unless
+explicitly contradicted below). Any subject-specific derivatives should be
+housed within each subject’s directory; if session-specific derivatives are
+generated, they should be deposited under a session subdirectory within the
+corresponding subject directory; and so on.
+
+In addition pipelines may generate derivatives that do not strictly belong to
```

Comment:
How are subgroups to be dealt with? E.g. patients versus controls. Perhaps like this:

    group-patients
    group-controls

where all subjections would be

    group-all

Reply:
I realized that none of our specific data types have templates or examples compatible with "group" outputs. I am tempted to remove this paragraph and leave covering "group" outputs to future PRs. Please let me know what do you think.

Reply:
Yeah, I'm happy to move `group` to a later PR. It should probably be thought out more carefully than we currently have bandwidth for.